src/FNA3D_Driver_Vulkan.c

```diff
@@ -3977,7 +3993,55 @@ FNA3D_Renderbuffer* VULKAN_GenColorRenderbuffer(
 	int32_t multiSampleCount,
 	FNA3D_Texture *texture
 ) {
-	/* TODO */
+	FNAVulkanRenderer *renderer = (FNAVulkanRenderer*) driverData;
+	VulkanTexture *vlkTexture = (VulkanTexture*) texture;
+	SurfaceFormatMapping surfaceFormatMapping = XNAToVK_SurfaceFormat[format];
+
+	VkImageViewCreateInfo imageViewInfo = {
+		VK_STRUCTURE_TYPE_IMAGE_VIEW_CREATE_INFO
+	};
+
+	VkExtent2D dimensions;
+	dimensions.width = width;
+	dimensions.height = height;
+
+	imageViewInfo.image = vlkTexture->handle;
+	imageViewInfo.viewType = VK_IMAGE_VIEW_TYPE_2D;
+	imageViewInfo.format = surfaceFormatMapping.formatColor;
+	imageViewInfo.components = surfaceFormatMapping.swizzle;
+	imageViewInfo.subresourceRange.aspectMask = VK_IMAGE_ASPECT_COLOR_BIT;
+	imageViewInfo.subresourceRange.baseMipLevel = 0;
+	imageViewInfo.subresourceRange.levelCount = 1;
+	imageViewInfo.subresourceRange.baseArrayLayer = 0;
+	imageViewInfo.subresourceRange.layerCount = 1;
+
+	/* Create and return the renderbuffer */
+	VulkanRenderbuffer *renderbuffer = SDL_malloc(sizeof(VulkanRenderbuffer));
+	renderbuffer->handle.image = vlkTexture->handle;
+	renderbuffer->handle.memory = NULL;
+	renderbuffer->handle.dimensions = dimensions;
+	renderbuffer->samples = XNAToVK_SampleCount(multiSampleCount);
+
+	VkResult result = renderer->vkCreateImageView(
+		renderer->logicalDevice,
+		&imageViewInfo,
+		NULL,
+		&renderbuffer->handle.view
+	);
```

Comment:
is there a particular reason we're not using the CreateImage function here?

Reply:
note to future self: it's because we are not actually creating a new Image, only a new ImageView for the Image contained in the texture that gets passed in